pkg/notes/document.go

```diff
@@ -109,6 +110,22 @@ func CreateDocument(notes ReleaseNotes, history ReleaseNotesHistory) (*Document,
 	return doc, nil
 }
 
+// RenderMarkdownTemplate renders a document using the Go template in `goTemplate`.
+func RenderMarkdownTemplate(doc *Document, bucket, tars, prevTag, newTag, goTemplate string) (string, error) {
```

Comment:
Why does `RenderMarkdownTemplate` take `bucket`, `tars`, `newTag` when it does not use them?

Reply:
I can strip it out.